src/stratis_cli/_actions/_logical.py

```diff
@@ -98,6 +98,7 @@ def list_volumes(namespace):
         List the volumes in a pool.
         """
         # pylint: disable=import-outside-toplevel
+        from ._constants import FILESYSTEM_INTERFACE
```

Comment:
Please move this import up to top of file, with other imports not from ```_data.py```, and same with similar.